### ISSUE TITLE
video orientation when recorded on smartphone

### ISSUE THREAD

**Issue by RamonSouza96** [MagicFoundation/Alcinoe#228]
hello @Zeus64 I would like to play a video with 9:16 aspect ratio on ALVideoPlayerSurface1, but when playing it doesn't leave the video in the correct orientation, I've studied the component a lot but I haven
't found a solution, could you give me a way to solve it ?
below is a print
![Sem título](https://user-images.githubusercontent.com/34917070/146816903-a1843856-c4ae-4a22-b00e-d59f3cdc081a.png)

**Comment by RamonSouza96** [comment]
![d1595ad5-1af4-4866-8c68-28619a4e75fc](https://user-images.githubusercontent.com/34917070/146821905-7e324c02-74d4-4682-aabd-245f0fdddf6e.jpg)

**I went deeper into the problem, and apparently found an android error.**

While recording a video on android in the 9:16 vertical position after saving, I noticed that the file width and height information are reversed.

The problem is not with midea player, but with the file proportion information.

A second doubt of mine is how other video players manage to reproduce in the correct proportion?
